### PR TITLE
Trying new appveyor configuration

### PR DESCRIPTION
**Pull request by xarkes**

### CODE REVIEW

**Comment by XVilka** [comment]
@xarkes why do you build against capstone release?

**Comment by xarkes** [comment]
It's a fail because I did some other build then I commited with -a. Will fix it thx

**Comment by xarkes** [comment]
Ok so with this configuration mingw32 and msvc (with meson) works great. You can download the artifacts from appveyor.

File: sys/cccl [hunk 1]
```diff
@@ -261,9 +265,9 @@ EOF
         ;;
 
     /*)
-        # All '/' options are assumed to be for cl and are passed through
-        val=$(echo "$1" | cut -c2-)
-        clopt="${clopt},${slash}${val}"
+        # All '/' options are assumed to be absolute path, not options
+        path=$(cygpath -aw "$1")
```
Comment (XVilka) [inline sys/cccl]: Not sure if this is a portable option... E.g. pure mingw installation wont have this

Comment (XVilka) [inline sys/cccl]: It's OK for now, just may be nice to keep that in mind.

**Comment by XVilka** [comment]
LGTM

**Comment by radare** [comment]
its green\o/

**Comment by radare** [comment]
ok to merge?

**Comment by xarkes** [comment]
Why not ? :)

**Pull request merged**